easybuild/easyblocks/o/openbabel.py

```diff
@@ -26,16 +26,30 @@
 EasyBuild support for OpenBabel, implemented as an easyblock
 
 @author: Ward Poelmans (Ghent University)
+@author: Oliver Stueker (Compute Canada/ACENET)
 """
 import os
 from easybuild.easyblocks.generic.cmakemake import CMakeMake
+from easybuild.framework.easyconfig import CUSTOM
 from easybuild.tools.modules import get_software_root, get_software_version
 from easybuild.tools.systemtools import get_shared_lib_ext
-
+from distutils.version import LooseVersion
 
 class EB_OpenBabel(CMakeMake):
     """Support for installing the OpenBabel package."""
 
+    @staticmethod
+    def extra_options():
+        extra_vars = {
+            'try_python': [True, "Try to build Open Babel's Python bindings. (-DPYTHON_BINDINGS=ON)", CUSTOM],
```

Comment:
> it suffices to just exclude the use of Python as a dependency?
Unfortunately that does not work if `Eigen` has been built with a Python dependency as OpenBabel will inherit it indirectly.